editor.planx.uk/src/@planx/components/SetFee/utils.ts

```diff
@@ -0,0 +1,98 @@
+import { PassportFeeFields } from "@opensystemslab/planx-core/types";
+import { Store } from "pages/FlowEditor/lib/store";
+
+import { PAY_FN } from "../Pay/model";
+import { DEFAULT_PAYMENT_PROCESSING_PERCENTAGE, VAT_PERCENTAGE } from "./model";
+
+type HandleSetFees = (params: {
+  passport: Store.Passport;
+  applyCalculatedVAT: boolean;
+  fastTrackFeeAmount: number;
+  applyServiceCharge: boolean;
+  serviceChargeAmount: number;
+  applyPaymentProcessingFee: boolean;
+}) => Store.Passport["data"];
+
+// SetFee outputs any `number` type PassportFeeFields, never `boolean` ones
+type OutputFees = Omit<
+  PassportFeeFields,
+  | "application.fee.exemption.disability"
+  | "application.fee.exemption.resubmission"
+  | "application.fee.reduction.alternative"
+  | "application.fee.reduction.parishCouncil"
+  | "application.fee.reduction.sports"
+>;
```

Comment:
`type-fest` has a helper for this that I've never really needed to reach for yet which also handles this.

```ts
type OutputFees = ConditionalPick<PassportFeeFields, number>;
```

Reply:
Handy! Updated :heavy_check_mark: